integrasjon/saf-klient/src/main/resources/saf/dokumentoversiktFagsakQuery.graphql

```diff
@@ -0,0 +1,38 @@
+query ($fagsakId: String!, $fagsaksystem: String!) {
```

Comment:
Trengs alle feltene i dette queryet - kan det forenkles?

Reply:
Man kan forenkle dersom man vil fortsette paradigme som ble brukt mot Joark før med to spørringer:
1. dokumentoversiktFagsakQuery: Hente opp oversikt over journalposter
2. journalpostQuery: Hente opp spesifikk journalpostId fra journalpost(er) i steg 1

Her returnerer jeg det samme for begge, og dermed kan man droppe å gjøre det i to steg. Videre blir JSON-klassene fylt med samme feltverdier, uavhengig av hvilken spørring. Velger å la det forbli slik.